modules/installing-ove-advantages.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/installing-ove.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-installing-ove-advantages_{context}"]
+= Installation method advantages
+
+This method has the following advantages for users who want to install a cluster primarily to run virtualized workloads using {VirtProductName}:
```

Comment:
Out of scope for merge review, but there are references to "this method" throughout, that it would be better to clarify what "this method" is. Especially something like at the beginning of this module, which doesn't have prior context set.